Sharing/ViewModels/ViewModel.swift

```diff
@@ -9,6 +9,12 @@ import OSLog
 
 final class ViewModel: ObservableObject {
 
+    // MARK: - Error
+
+    enum ViewModelError: Error {
+        case unknown
```

Comment:
Since we only use this in one place, how about a slightly more descriptive name here? Hopefully it's never used, but maybe something simple like:

```suggestion
        case invalidRemoteShare
```